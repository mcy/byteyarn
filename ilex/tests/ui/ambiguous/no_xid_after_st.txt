q'xyz'q %'xyz'qua