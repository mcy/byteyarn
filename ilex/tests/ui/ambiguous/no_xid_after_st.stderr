error: extraneous characters after `%'...'q`
 --> ambiguous/no_xid_after_st.txt:1:16
  |
1 | q'xyz'q %'xyz'qua
  |                ^^
  |               -- help: maybe you meant to include a space here
  |
  = note: reported at: ilex/src/rt/emit2.rs:779:10

error: aborting due to previous error
